### ISSUE TITLE
Auto deploy using Travis CI

### ISSUE THREAD

**Issue by jasny** [legalthings/lto-api.java#32]
Copy the Travis configuration from the python library.

- It runs the tests and then releases a new version on GitHub when `master` is updated. The automatically creates a new tag.
- On a new tag the library should be released to Maven.

By adding `[bump:minor]` or `[bump:major]` to a commit message you can bump the minor or major version number.

**Comment by KaloyanTanev** [comment]
And for patch versions? Do we increment this one on every PR to master?

**Comment by jasny** [comment]
Yes, by default a new patch release is done 9n every merge on master.
